### ISSUE TITLE
Unrecognized Code

### ISSUE THREAD

**Issue by BlueRadley** [xvwyh/BuildPad#12]
It seems that, with the last patch, build codes got updated, and BuildPad gives "Unrecognized Code" when trying to add the new style of code.

For example, the code for my thief build is:
Pre-Patch:    [&DQUcGyw1BxmUEgwBVQFVAS8BLwELAVcBKAAoAAAAAAAAAAAAAAAAAAAAAAA=]
Post-Patch:  [&DQUcGyw1BxmUEgwBVQFVAS8BLwELAVcBKAAoAAAAAAAAAAAAAAAAAAAAAAABLwAA]

Pre-Patch was copied from BuildPad, post-patch from my build tab in-game.

**Comment by xvwyh** [comment]
Yeah, the new patch added support for storing the weapon types used in the build + weapon skill customizations (that will first appear in game with SotO and were in the beta), and the client now generates chatlinks with this new structure even for non-beta characters. I'll need to add support for that. I just need to overcome my laziness...

Old codes should still work in-game, and you can make the build directly in BuildPad, so that's the workaround for now.

**Comment by BlueRadley** [comment]
No worries, thank you and thanks for the add-on. It’s been an absolute godsend.

**Comment by xvwyh** [comment]
I implemented everything that's needed for the new codes to work and be editable. One hurdle that remains before I can release the update, tho, is that I wanted to get rid of my autoupdate mechanism and switch to using arcdps's own autoupdater for extensions/plugins, which was added to it semi-recently. Trying to make sure that the transition goes smoothly and various messy edge cases are accounted for.
